Cargo.toml

```diff
@@ -65,3 +74,8 @@ esp-idf-sys = { version = "0.35", features = ["binstart"] }
 mipidsi = "0.5.0"
 display-interface-spi = "0.4.1"
 embedded-graphics = "0.7.1"
+
+[[package.metadata.esp-idf-sys.extra_components]]
+remote_component = { name = "onewire_bus", version = "^1.0.2" }
+bindings_header = "bindings.h"
```

Comment:
I merged the onewire_bus.h into esp-idf-sys. You can now remove this header stuff here and add a [patch.crates-io] section to poiint esp-idf-hal to espi.df-sys git master branch so you can test it out. E.g
```toml
[patch.crates-io]
esp-idf-sys = { git = "https://github.com/esp-rs/esp-idf-sys"}

```

Reply:
Done. Will remove the patch towards the end.